scripts/remove-cosmos-test-registrations.js

```diff
@@ -0,0 +1,67 @@
+const {
+  establishConnectionToCosmos,
+  closeCosmosConnection
+} = require("../src/connectors/cosmos.client");
+const { connectToDb, closeConnection } = require("../src/db/db");
+const {
+  getRegistrationByFsaRn
+} = require("../src/connectors/registrationDb/registrationDb");
+const { logEmitter } = require("../src/services/logging.service");
+
+let successfulDeletions = [];
+let failedDeletions = [];
+let beCache;
+
+const deleteTestRegistrationsFromCosmos = async () => {
+  beCache = await establishConnectionToCosmos("registrations", "registrations");
+
+  const records = await beCache.find({}).toArray();
+
+  await connectToDb();
+
+  // Find records that aren't in PG (test records) and remove them from Cosmos.
+  const promises = records.map(async (rec) => {
+    const registration = await getRegistrationByFsaRn(rec["fsa-rn"]);
+    if (!registration) {
+      await removeCosmosRecord(rec);
+    }
+  });
+  await Promise.all(promises);
+  logEmitter.emit(
+    "info",
+    `Successfully deleted ${successfulDeletions.length} registrations from Cosmos: ${successfulDeletions}`
+  );
+  logEmitter.emit(
+    "info",
+    `Failed to delete ${failedDeletions.length} registrations from Cosmos: ${failedDeletions}`
+  );
+  return;
+};
+
+const removeCosmosRecord = async (record) => {
+  logEmitter.emit("info", `Removing BE Cache FSA-RN: ${record["fsa-rn"]}`);
+  try {
+    await beCache.deleteOne({ "fsa-rn": record["fsa-rn"] });
+    successfulDeletions.push(record["fsa-rn"]);
```

Comment:
I did initially output them but thought it might be a little too much logging, but then again there's already going to be a lot and like you said would be good to have a record so I'll implement this.

Reply:
Changed this to just output those remaining after the script that failed. Otherwise it'll output 45K registration numbers for some of the scripts